src/js/stores/preferences.js

```diff
@@ -99,6 +99,10 @@ define(function (require, exports, module) {
             return this._preferences;
         },
 
+        get: function (key, unsetValue) {
+            return this._preferences.get(key, unsetValue);
+        },
+
```

Comment:
I'm the guilty party here. I thought it was really useful though, because otherwise the only state we get from the preferences store is just the map, which we only use to call `get` on. It seemed like a fair way to save a step... But, if you don't like it, @DivyaPrabhakar can rip it out!